README.md

```diff
@@ -116,16 +130,30 @@ relationships between the two are set up in Active Record.
 Write code to do the following:
 
 1. Create a lamp with 3 wishes remaining and a genie named 'Genie'
+
 2. Create a relationship between 'Genie' and the lamp.
+<!-- This relationship will go in the model files for each entity.  -->
+
 3. Update the lamp so it only has one wish left.
   * Oh no... Jafar has Aladdin! Thankfully he's wished to become a genie himself!
 4. Create a new Genie named 'Jafar' and put him in a new lamp with 3 wishes left.
 5. Free the good Genie by setting his lamp to `nil`
 
-
 Write your code here:
 ```ruby
 # code here
+lamp = Lamp.create(wishes_remaining: 3)
+genie = Genie.create(name: "Genie")
+class Genie ActiveRecord::Base
+belongs_to:lamp
+end
+class Lamp ActiveRecord::Base
+has_one: genie
+end
```

Comment:
Here I was looking for the specific relationship between the genie you created and the lamp, e.g.:

```ruby
lamp.genie = genie
lamp.save
```